sre-recipes/recipe.py

```diff
@@ -70,11 +70,8 @@ def _run_command(command):
     @staticmethod
     def _run_command_interactive(command):
         """Runs the given interactive command that waits for user input and returns any output and error"""
-        process = subprocess.Popen(
-            command.split(), shell=True, stderr=subprocess.PIPE
-        )
-        output, error = process.communicate()
-        return output, error
+        process = subprocess.run(command.split())
```

Comment:
nit: It might be useful to return some data from the completed process. If not, then you should get rid of the unused process variable